twoliter/embedded/repack.Dockerfile.dockerignore

```diff
@@ -1,4 +0,0 @@
-/
```

Comment:
Maybe it's just a git thing, but this file delete is in the commit after the one that removed it from Twoliter's embedding.